templates/pricing/pro.html

```diff
@@ -226,6 +226,8 @@ <h5>Notes</h5>
       <hr class="u-no-margin--bottom">
       <p>N.B.: Weekday support is half the price of 24/7 support</p>
       <hr class="u-no-margin--bottom">
+      <p>N.B.: Subscriptions are not just for enterprise customers. Anyone can get <a href="/pro">a personal subscription</a> for free on up to 5 machines.</p>
```

Comment:
```suggestion
      <p>N.B.: Subscriptions are not just for enterprise customers. Anyone can get <a href="/pro">a personal subscription</a> for free on up to 5 machines</p>
```
Removed the period on the doc to maintain consistency